lutra/bindings/python/noxfile.py

```diff
@@ -0,0 +1,42 @@
+"""Nox session configuration."""
+import os
+from pathlib import Path
+from typing import List
```

Comment:
To be honest, I don't know what does this file do.

Installs different versions of Python and runs the tests? But implemented in Python?

precommit mypy seems to have a problem with there being two noxfile.py, but if my understanding is correct, it would make sense to have two, as these are two separate packages.

It would make more sense to run mypy separately on prqlc-python and lutra-python

Reply:
Some people really like `nox`. I haven't found it particularly helpful in my work.

So unless someone remembers that we really needed it in `prql_python`, I would suggest deleting it in Lutra...